Makefile

```diff
@@ -72,6 +72,12 @@ nightly-debug:
 	cargo build -p genesis-populate --features nearcore/nightly,nearcore/performance_stats,nearcore/memory_stats
 
 
+assertions-release: NEAR_RELEASE_BUILD=release
+assertions-release:
+	CARGO_PROFILE_RELEASE_DEBUG=true CARGO_PROFILE_RELEASE_DEBUG_ASSERTIONS=true cargo build -p neard --release --features performance_stats,memory_stats
+	cargo build -p store-validator --release --features nearcore/performance_stats,nearcore/memory_stats
```

Comment:
Do we actually need to build store validator as well? 


Do we need perf_stats here? Debug assertions might skew perfomance.

Reply:
> store_validator

Not really, removed.

> perf_stats

That is a good question, and I don't know the answer. I suggest to start with `perf_stats` to be consistent with the canary instances we already run.

Reply:
cc https://github.com/near/nearcore/issues/6957